Assets/Scripts/Actors/PlayerActor.cs

```diff
@@ -7,22 +7,34 @@ public class PlayerActor : Actor
 	ActorCamera _actorCamera = null;
 	PlayerActorStats _actorStats = null;
 	PlayerActorInventory _playerActorResources = null;
+	PlayerControls _controls = null;
+	Cutting _cutting = null;
 
 	public ActorCamera actorCamera
 	{
 		get { return _actorCamera; }
 	}
 
-	public PlayerActorInventory actorResources
+	public PlayerActorInventory inventory
 	{
 		get { return _playerActorResources; }
 	}
 
-	public PlayerActorStats actorStats
+	public PlayerActorStats stats
```

Comment:
We can probably rename these to PlayerInventory and PlayerStats since they're specific to the player.